sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/specialized/BlockBlobAsyncClient.java

```diff
@@ -94,12 +95,12 @@ public final class BlockBlobAsyncClient extends BlobAsyncClientBase {
     /**
```

Comment:
It looks like `MAX_UPLOAD_BLOB_BYTES_LONG` did not get updated here to use the new constants class.